package.json

```diff
@@ -102,5 +90,9 @@
       "<rootDir>/query-planner-js",
       "<rootDir>/subgraph-js"
     ]
+  },
+  "volta": {
```

Comment:
I'm interested to know more about this. Is it staying? What do we use it for?

Reply:
so volta is what i use to manage my node runtime versions across a bunch of different projects. i'd love to keep it and it's helpful for folks who use volta and will just be ignored by folks who don't use it.